### PR TITLE
bump react-native

### PR DESCRIPTION
**Pull request by Dinek007**

### CODE REVIEW

**Review by holmesworcester — APPROVED** [review]

File: packages/mobile/package.json [hunk 1]
```diff
@@ -37,9 +37,9 @@
     "nodejs-mobile-react-native": "^0.8.0",
     "pkijs": "*",
     "pvutils": "^1.0.17",
-    "react": "17.0.1",
+    "react": "18.0.0",
```
Comment (siepra) [inline packages/mobile/package.json:40]: Can we confirm this is safe for the rest of the project?

Comment (Dinek007) [reply packages/mobile/package.json:40]: hmm its package.json in mobilce, should not have a impact on the rest of the project

**Review by siepra — APPROVED** [review]

**Review by EmiM — APPROVED** [review]

**Comment by siepra** [comment]
I just saw this issue https://github.com/nodejs-mobile/nodejs-mobile-react-native/issues/22
We should hold until we make sure our app works after bumping rn version

File: packages/mobile/android/gradle.properties [hunk 2]
```diff
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx4608m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```
Comment (kowalski) [inline packages/mobile/android/gradle.properties:13]: it's weird to not use power of 2 in this context.. not that it matters... but I'd put `4096m`

**Review by kowalski — APPROVED** [review]

**Pull request merged**